fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -566,6 +566,7 @@ struct TenantManagementWorkload : TestWorkload {
 					// Update our local tenant state to include the newly created one
 					self->maxId = entry.get().id;
 					TenantData tData = TenantData(entry.get().id, tenantItr->second.tenantGroup, true);
+					tData.tenant->name = tenantItr->first;
```

Comment:
The `Tenant` object has a constructor that takes the name, so we can use that for conciseness.